1_html-css/index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Niloufar portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="media-queries.css">
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+</head>
+
+<body>
+    <nav id="desktop-nav">
+        <div class="logo">Niloufar Khaefi</div>
+        <div>
+            <ul class="nav-links">
+                <li><a href="#about">About</a></li>
+                <li><a href="#skills">Skills</a></li>
+                <li><a href="#projects">Projects</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+    <header id="profile">
+        <div class="profile-pic">
+            <img src="./assets/profile-niloufar.jpg" alt="">
+        </div>
+        <div class="profile-txt">
+            <p>Hello, I'm</p>
+            <h2>Niloufar Khaefi!</h2>
+            <h4>A Joniur Forntend Developer</h4>
+            <div class="profile-btn">
+                <button class="btn cv-btn" onclick="window.open('./assets/Niloufar-cv.pdf')">Download CV</button>
+                <button class="btn contact-btn" onclick="location.href = '#contact'">Contact Info</button>
+            </div>
+            <div class="SoMe-links">
+                <a href="https://github.com/Niloufar97"><img src="assets/square-github.svg" alt=""></a>
+                <a href="https://www.linkedin.com/in/niloufar-khaefi/"><img src="assets/linkedin.svg" alt=""></a>
+            </div>
+        </div>
+    </header>
+    <section id="about" data-aos="fade-up" data-aos-duration="2000">
+        <div class="about-title">
+            <p>Get To Know More</p>
+            <h2 class="big-font">About Me</h2>
+        </div>
+        <div class="about-container">
+            <div class="about-pic">
+                <img src="./assets/undraw_dev_focus_re_6iwt.svg" alt="">
+            </div>
+            <div class="about-txt">
+                <div class="experience-box">
+                    <div class="experience">
+                        <h5>Experience</h5>
+                        <p>1+ year</p>
+                        <p>Frontend Development</p>
+                    </div>
+                    <div class="education">
+                        <h5>Education</h5>
+                        <p>B.Sc. Bachlors Degree</p>
+                        <p>Isfahan University of Technology </p>
+                    </div>
+                </div>

```

Comment:
Instead of having a div as a container 'experience-box', use the semantic tag ul, because you want to have two entities with the same type of content inside.
Two dives  'experience' and 'education', I would change to the li tag accordingly.